### PR TITLE
make faq component

### PR DESCRIPTION
**Pull request by andreidimaano**
### Description

- put some _gud_ placeholder text
- working accordion with _some_ styling

https://user-images.githubusercontent.com/56818175/180634372-af772491-0087-49ed-adee-07461313a79a.mp4

### Testing Plan
`yarn format`

Screen Shot 2022-07-23 at 10 55 08 PM

### CODE REVIEW

**Comment by andreidimaano** [comment]
me when submitting a PR:
image

me when PR is accepted: 
image

**Comment by minsooerickim** [comment]
> ### Issue #8: make FAQ section
> ### Description
> 
>     * put some _gud_ placeholder text
> 
>     * working accordion with _some_ styling
> 
> 
> accordion.mp4
> ### Testing Plan
> 
> `yarn format`
> 
> Screen Shot 2022-07-23 at 10 55 08 PM

this looks pretty cleannnnnnnnnnn

**Comment by minsooerickim** [comment]
> ### Issue #8: make FAQ section
> ### Description
> 
>     * put some _gud_ placeholder text
> 
>     * working accordion with _some_ styling
> 
> 
> accordion.mp4
> ### Testing Plan
> 
> `yarn format`
> 
> Screen Shot 2022-07-23 at 10 55 08 PM

I did not know `yarn format` was a thing. I'm curious how this works, cuz it doesn't seem like it didn't adhere to the prettier.rc I defined in root. I wonder if it's formatting by "default" prettier settings or something cuz I specified no semicolons in the .rc file but it looks like it autoformatted to add the semicolons in ur files

**Comment by minsooerickim** [comment]
> me when submitting a PR: image
> 
> me when PR is accepted: image

lmaoooo, I feel so much power reviewing prs

**Comment by andreidimaano** [comment]
> 

i added yarn format in the package.json and ran a script, but it's pretty weird that it didn't follow the .prettierrc 이상하죠...

**Comment by minsooerickim** [comment]
> > 
> 
> i added yarn format in the package.json and ran a script, but it's pretty weird that it didn't follow the .prettierrc 이상하죠...

it looks like u have to specify the path of the rc file :)

**Comment by minsooerickim** [comment]
There should probably be a max width or margins for the accordion. I think its way too wide for desktop view
image

**Comment by andreidimaano** [comment]
> There should probably be a max width or margins for the accordion. I think its way too wide for desktop view image

that should be controlled by the page layout - probably use a grid

**Comment by minsooerickim** [comment]
> e page layout - probably use a grid

when you say page layout what are you referring to the most parent element so in like `index.tsx`?

**Comment by minsooerickim** [comment]
> > e page layout - probably use a grid
> 
> when you say page layout what are you referring to the most parent element so in like `index.tsx`?

so if we do that, that will affect everything on the page, so I feel like we shouldn't do that. idk I feel like the margins should go within the faq component itself?

**Comment by andreidimaano** [comment]
> > > e page layout - probably use a grid
> > 
> > 
> > when you say page layout what are you referring to the most parent element so in like `index.tsx`?
> 
> so if we do that, that will affect everything on the page, so I feel like we shouldn't do that. idk I feel like the margins should go within the faq component itself?

that's true, it would affect everything if we do the most parent element.

instead of the most parent element, it's just the parent of the FAQ component, so when we start implementing the overall layout of the page, we can fix the sizing of the FAQ component — the FAQ component is only really big right now because it takes up 100% of the width of the parent which is a good thing 😄

**Comment by minsooerickim** [comment]
> > > > e page layout - probably use a grid
> > > 
> > > 
> > > when you say page layout what are you referring to the most parent element so in like `index.tsx`?
> > 
> > 
> > so if we do that, that will affect everything on the page, so I feel like we shouldn't do that. idk I feel like the margins should go within the faq component itself?
> 
> that's true, it would affect everything if we do the most parent element.
> 
> instead of the most parent element, it's just the parent of the FAQ component, so when we start implementing the overall layout of the page, we can fix the sizing of the FAQ component — the FAQ component is only really big right now because it takes up 100% of the width of the parent which is a good thing 😄

ah okay I see what your saying, sounds good to me!

File: components/FAQ/Faq.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            question={faqItem.question}
+            answer={faqItem.answer}
+          />
+        )
+      })}
+    </div>
+  )
+}
+
+export default Faq
```
Comment (minsooerickim) [inline components/FAQ/Faq.tsx:23]: can you export it in line with the declaration just to keep consistent with the project

Comment (minsooerickim) [reply components/FAQ/Faq.tsx:23]: i know some components are exported at the bottom..that's our bad but we've been meaning to export it in line for everything

**Pull request merged**